### DISCUSSION TITLE
Cannot push images to the docker-registry using the kubectl plugin

### DISCUSSION THREAD

**Discussion by Dean063099** [moby/moby#46172] [Q&A]
I'm a fresher when it comes to using k8s. Please help me with the issue I am facing. I am trying to push the images to the local docker-registry. Some of the images are being pushed successfully, but in a couple of seconds it is failing. It is giving me this error message:

```json lines
{"status":"The push refers to repository [docker-registry:30001/netapp/astracc/22.08.1-26/graphql-server]"}
{
  "errorDetail":{"message":"Get \"https://docker-registry:30001/v2/\": dial tcp 192.168.0.96:30001: connect: connection refused"},
  "error":"Get \"https://docker-registry:30001/v2/\": dial tcp 192.168.0.96:30001: connect: connection refused"
}
```
I tried checking if my docker-registry is running. This is the output to check if my registry is up and running:

```
[root@kubmas2 acc]# docker ps | grep registry

ab05d3e41abf registry "/entrypoint.sh /etc…" About a minute ago Up About a minute k8s_registry_docker-registry-547576f64c-9w8rl_registry_b48428e8-4b45-422f-906c-37c6c6286f80_0

d59cab9271fc k8s.gcr.io/pause:3.6 "/pause" About a minute ago Up About a minute k8s_POD_docker-registry-547576f64c-9w8rl_registry_b48428e8-4b45-422f-906c-37c6c6286f80_0

f40fdeae0b7c registry.k8s.io/sig-storage/csi-node-driver-registrar "/csi-node-driver-re…" 8 days ago Up 8 days k8s_driver-registrar_trident-csi-nxhxb_trident_6245b9f7-205a-463a-bb7b-0d15a08b5189_0
```

**Comment by thaJeztah** [comment]
How is the registry started? Does it have ports mapped? Is it accessible from the host where the daemon runs at `192.168.0.96:30001`?

**Reply by Dean063099** [reply]
It was created using a script we got from one of the github repositories here. It have ports mapped which are 30001 and 5000. I have tried disabling the firewall and also tried it with the firewall enabled with both ports open. 

Could you elaborate this question 'Is it accessible from the host where the daemon runs at 192.168.0.96:30001?'
